arangod/Utils/Cursor.h

```diff
@@ -85,6 +91,18 @@ class Cursor {
 
   void setDeleted() { _isDeleted = true; }
 
+  void setLastQueryBatchObject(velocypack::Builder& builder) {
+    _currentBatchResult.second = builder;
+  }
```

Comment:
This API is a bit unintuitive, because we are passing the Builder by mutable reference, but are then copying it anyway.